bundles/binding/org.openhab.binding.onewire/src/main/java/org/openhab/binding/onewire/internal/connection/OneWireConnection.java

```diff
@@ -81,91 +94,155 @@ public static synchronized OwfsConnection getConnection() {
 
     /**
      * Tries to connect either by IP or serial bus, depending on supplied config data.
-     * 
+     *
      * @return true if connection was established, false otherwise
      */
     public static synchronized boolean connect() {
-        if (cvIp != null && cvPort > 0) {
-            OwfsConnectionFactory owfsConnectorFactory = new OwfsConnectionFactory(cvIp, cvPort);
-            OwfsConnectionConfig owConnectionConfig = new OwfsConnectionConfig(cvIp, cvPort);
-            owConnectionConfig.setTemperatureScale(cvTempScale);
-            owConnectionConfig.setPersistence(OwPersistence.ON);
-            owConnectionConfig.setBusReturn(OwBusReturn.ON);
-            owfsConnectorFactory.setConnectionConfig(owConnectionConfig);
+        OwfsConnectionFactory owfsConnectorFactory = new OwfsConnectionFactory(cvIp, cvPort);

```

Comment:
Just to confirm that the `cvIp != null` check is no longer needed before this line?